src/components/Button/Button.tsx

```diff
@@ -29,6 +29,7 @@ export interface Props {
   processing?: boolean;
   id?: string;
   className?: string;
+  style?: CSSProperties;
```

Comment:
Just curious why this addition? I like the idea for sure, we can move away from className/makeStyles.

Reply:
Yes, to move away from className and makeStyles. In dev mode with react 18 some styles are not working as expected because of the use of makeStyles

Reply:
Got it, thanks!